src/Stryker.Core/Stryker.Core/Baseline/AzureBaselineProvider.cs

```diff
@@ -0,0 +1,179 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Stryker.Core.Logging;
+using Stryker.Core.Options;
+using Stryker.Core.Reporters.Json;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stryker.Core.Baseline
+{
+    public class AzureBaselineProvider : IBaselineProvider
```

Comment:
Looking at this again it should probably be renamed to AzureFileShareBaselineProvider to avoid confusion.

Reply:
Applied.